apps/src/templates/rubrics/RubricContainer.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/order */
```

Comment:
This is because some of the imports had a comment and were purposefully grouped.

I added a comment and also ran the order linter before re-adding the groupings.